tests/h/search/index_test.py

```diff
@@ -349,6 +349,23 @@ def test_it_indexes_specific_annotations(self, batch_indexer, es_client, factori
                                             doc_type="annotation",
                                             id=id)
 
+    def test_it_does_not_index_deleted_annotations(self, batch_indexer, es_client, factories):
+        ann = factories.Annotation()
+        # create deleted annotations
+        ann_del = factories.Annotation(deleted=True)
+
+        batch_indexer.index()
+
+        result_indexed = es_client.conn.get(index=es_client.index,
```

Comment:
We are duplicating these lines to get an annotation from the index by ID in a few tests. I would be inclined, perhaps as a separate change once these BatchIndexer tests are merged, to re-use the `get` fixture from the indexing test.